app/modules/sightings/models.py

```diff
@@ -26,6 +28,28 @@ class SightingStage(str, enum.Enum):
     failed = 'failed'
 
 
+# # A sighting may have multiple IaConfigs, each with multiple algorithms, even if only one is supported for MVP
```

Comment:
Don't worry about this stuff either, this is going to be a JSON Blob once PR217 is in